### DISCUSSION TITLE
Is it possible to have timed responses?

### DISCUSSION THREAD

**Discussion by alacamda** [dialogic-godot/dialogic#1983] [General]
Hi there! I recently started working with Dialogic and it's great! I have a question though: is it possible to add timed responses/options?  I mean different options the player can choose during a few seconds. If the player does not choose any option in the given time, the conversation moves on

Thank you!

**Comment by Pheubel** [comment]
i think that it should be possible, one the top of my head, you could maybe create your own event for a timed choice that communicates with the choice subsystem. when the event starts it would tell the subsystem about the choices that can be made. it would also start a timer that when expired will automatically select one of the possible choices or a fallback option. you might have to roll your own version of the choice subsystem, but whoever way, i would recommend you look at [subsystem_choices.gd](https://github.com/dialogic-godot/dialogic/blob/main/addons/dialogic/Modules/Choice/subsystem_choices.gd).

**Reply by alacamda** [reply]
OK thanks @Pheubel. I will take a look, though I still think it would be nice to have this option implemented as a standard event in dialogic

**Reply by Pheubel** [reply]
you can always make an issue to suggest adding it to dialogic out of the box

**Reply by alacamda** [reply]
OK Thanks!
